protobufs/document/service.proto

```diff
@@ -80,3 +73,62 @@ message CreateDocumentProofForVersionRequest {
   string version = 3;
   repeated string fields = 4;
 }
+
+message RequestHeader {
+  repeated ReadAccess read_access = 1;
+  repeated WriteAccess write_access = 2;
+}
+
+enum ReadAction {
```

Comment:
can we read the actions from the centrifuge-protobufs instead of duplicating here them?

Reply:
yes, good call.